docs/content/en/docs/concepts/tracing-policy/selectors.md

```diff
@@ -682,11 +686,79 @@ This action uses the dedicated `argFd` and `argName` fields to get respectively
 the index of the file descriptor argument and the index of the name argument in
 the call.
 
+While the mapping between the file descriptor and file name remains in place
+(i.e. between `FollowFD` and `UnfollowFD` for the same file descriptor) tracing
```

Comment:
```suggestion
(that is, between `FollowFD` and `UnfollowFD` for the same file descriptor) tracing
```

note: we generally try to avoid use of latin abbreviations. See: https://docs.cilium.io/en/stable/contributing/docs/docsstyle/#don-t-use-latin-abbreviations.